### ISSUE TITLE
JS editor autocomplete on small files

### ISSUE THREAD

**Issue by iakov** [trikset/trik-studio#36]
When file is small/new one autocomplete does not work as expected. We can parse system.js to extend  dictionary or even use reflection to get publick slots/Q_INVOKABLE methods from object.

**Comment by iakov** [comment]
Probably it is fixed, already, for 19.04

**Comment by anastasiia-kornilova** [comment]
@iakov, can we close this issue?

**Comment by iakov** [comment]
Current state: extended autocompletion works, but shows methods only after the first letter is typed and the populated list is mixed with other tokens (type `brick().i` to see both `i2c` and `if` in the list). 
Can be postponed until next version.
